src/main/java/teammates/common/datatransfer/FeedbackSessionQuestionsBundle.java

```diff
@@ -137,4 +163,49 @@ public void resetAllResponses() {
             questionResponseBundle.put(question, new ArrayList<FeedbackResponseAttributes>());
         }
     }
+
+    public String getNameForEmail(String email) {
+        String name = emailNameTable.get(email);
+        if (name == null || name.equals(Const.USER_IS_MISSING)) {
+            return Const.USER_UNKNOWN_TEXT;
+        } else if (name.equals(Const.USER_IS_NOBODY)) {
+            return Const.USER_NOBODY_TEXT;
+        } else if (name.equals(Const.USER_IS_TEAM)) {
+            return getTeamNameForEmail(email);
+        } else {
+            return name;
+        }
+    }
+
+    public String getTeamNameForEmail(String email) {
+        String teamName = emailTeamNameTable.get(email);
+        if (teamName == null || email.equals(Const.GENERAL_QUESTION)) {
+            return Const.USER_NOBODY_TEXT;
+        }
+        return teamName;
+    }
+
+    public String appendTeamNameToName(String name, String teamName) {
+        String outputName;
+        if (name.contains("Anonymous") || name.equals(Const.USER_UNKNOWN_TEXT)
```

Comment:
Looks like @sujeet14108 is working on this issue now. Note that if that PR is merged before this one, then this will need to be modified accordingly, but for now let's leave it as it is.